clippy_lints/src/methods/or_fun_call.rs

```diff
@@ -52,16 +55,27 @@ pub(super) fn check<'tcx>(
 
             then {
                 let mut applicability = Applicability::MachineApplicable;
+                let hint = "unwrap_or_default()";
+                let mut shrink = span;
```

Comment:
This is really nit-picky, and I'm fine with keeping it like this. But you could give this span a better name, like: `sugg_span`.

Reply:
> This is really nit-picky, and I'm fine with keeping it like this. But you could give this span a better name, like: `sugg_span`.

Ok, next commit will change it.